src/structlog/processors.py

```diff
@@ -132,6 +99,86 @@ def __call__(
         )
 
 
+class LogfmtRenderer:
+    """
+    Render ``event_dict`` using the logfmt format.
+
+    :param sort_keys: Whether to sort keys when formatting.
+    :param key_order: List of keys that should be rendered in this exact
+        order. Missing keys are rendered with empty values, extra keys
+        depending on *sort_keys* and the dict class.
+    :param drop_missing: When ``True``, extra keys in *key_order* will be
+        dropped rather than rendered with empty values.
+
+    .. versionadded:: 21.5.0
+    """
+
+    def __init__(
+        self,
+        sort_keys: bool = False,
+        key_order: Optional[Sequence[str]] = None,
+        drop_missing: bool = False,
+    ):
+        self._ordered_items = _items_sorter(sort_keys, key_order, drop_missing)
```

Comment:
if you like, you could add an `if logfmt is None: raise ...` here (if you're willing to add a test too).

Reply:
Fixed by 44a19e6 (drop `logfmt` dependency).